src/shogun/distributions/Gaussian.cpp

```diff
@@ -267,7 +267,7 @@ float64_t CGaussian::compute_log_PDF(SGVector<float64_t> point)
 	return -0.5 * answer;
 }
 
-SGVector<float64_t> CGaussian::get_mean()
+SGVector<float64_t>& CGaussian::get_mean()
```

Comment:
What happens if you put the `get_mean()` output into a local variable first? It is a bit nicer to read anyways